source/typescript.txt

```diff
@@ -0,0 +1,35 @@
+.. _typescript-center:
+
+==========
+TypeScript
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+Welcome to the documentation site for official MongoDB TypeScript support.
+Since Typescript builds on JavaScript, you can use the
+`Node.js Driver <https://www.mongodb.com/docs/drivers/node/current/>`__
```

Comment:
S: This sounds better to me as "MongoDB Node.js Driver" in this sentence. There might already be a source constant in docs-node to this effect.

Reply:
I went ahead and created constants that match the constants in the node driver. Thanks for that idea!